packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -248,11 +266,72 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 		}
 	}
 
+	const resolvedUrl: IFluidResolvedUrl | undefined =
+		containerState.resolvedUrl as IFluidResolvedUrl;
+
 	return (
```

Comment:
Style question: Should we put the Container state and audience data above this information? Especially since they don't populate until we're attached, it might be nicer to have the state appear higher.

Reply:
Yeah, I like that idea. Will update.